x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/waterfall/waterfall_flyout.tsx

```diff
@@ -45,19 +55,21 @@ export function WaterfallFlyout({
           spanId={currentItem.id}
           parentTransactionId={parentTransactionId}
           traceId={currentItem.doc.trace.id}
-          onClose={() => toggleFlyout({ history })}
+          onClose={() => toggleFlyout({ history, flyoutDetailTab: undefined })}
```

Comment:
why do you need to pass `flyoutDetailTab: undefined` ?

I expect `toggleFlyout`will handle it as it handles the _missing_ item

Reply:
fixed [here](https://github.com/elastic/kibana/pull/147426/commits/9b273112a547e89356431fa7bf6202fde3557e72)